sdk/v1/client_test.go

```diff
@@ -468,7 +468,7 @@ func TestClient_PatchJob(t *testing.T) {
 
 			Convey("Then an empty ETag is returned", func() {
 				So(respHeaders, ShouldNotBeNil)
-				So(respHeaders, ShouldResemble, client.RespHeaders{ETag: ""})
+				So(respHeaders, ShouldResemble, &client.RespHeaders{ETag: ""})
```

Comment:
same as above, as respHeader is nil in error scenario

Reply:
Again, although respHeader is nil, it is an empty RespHeaders object that gets returned by the PatchJobs function.

Reply:
same ETag/Json to be nil in error scenario as per the comment above

Reply:
ok, I have changed this in line with the GetTask method as requested